src/accountTransactions.ts

```diff
@@ -51,6 +57,80 @@ export class SimpleTransferWithMemoHandler
     }
 }
 
+export class DeployModuleHandler
+    implements AccountTransactionHandler<DeployModulePayload>
+{
+    getBaseEnergyCost(payload: DeployModulePayload): bigint {
+        const cost: number = Math.round((payload.content.length + 1) / 10);
```

Comment:
Perhaps documenting where this comes from, and what it needs to be in sync with. Where is the `+1` from? The energy cost is size of the module in bytes / 10.